packages/alea-frontend/pages/api/job-portal/get-organization-id.ts

```diff
@@ -0,0 +1,22 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { checkIfGetOrSetError, executeDontEndSet500OnError } from '../comment-utils';
+
+export async function getOrganizationId(organizationName: string, res: NextApiResponse) {
```

Comment:
Fix this renaming issue everywhere
```suggestion
export async function getOrganizationIdOrSet500OnError(organizationName: string, res: NextApiResponse) {
```